crates/hc_bundle/src/cli.rs

```diff
@@ -51,6 +51,11 @@ pub enum HcDnaBundle {
         /// provided working directory.
         #[structopt(short = "o", long)]
         output: Option<PathBuf>,
+
+        /// Output shared object "dylib" files
+        /// that can be used to run this happ on iOS
+        #[structopt(short = "i", long)]
```

Comment:
Are there other possible reasons for outputting dylibs other than IOS support? If so maybe this should be called `-d` `--dylibs`

Reply:
Kinda feels like "dylib" should be in the name of this command but I'm not close enough to it to have a feel.

Given that we may support other dylib targets, does that change your choice of name? Imagine what other target names will look like. Wondering about `--dylib-ios`, `--dylib-raspberry-pi`, etc.

Also this feels too specific for a short name, `-i` for "iOS" seems a stretch.

Feel free to take any or none of these suggestions, I won't block on this.

Reply:
You can add in dylib, sure thing. And remove the short option.